spec.bs

```diff
@@ -2659,20 +2659,59 @@ algorithms to achieve the outcomes described in the above explanatory content.
   or <{fencedframe}>'s [=container policy=], when it is constructed.
 </div>
 
+<div algorithm>
+  Create a new algorithm, called <dfn export>Create a Permissions Policy for a fenced
```

Comment:
Let's rename this to "Derive permissions policy directly from fenced frame config instance" or something super explicit like that, as discussed offline.

Also: does this need to be exported?